controllers/databaseclusterbackup_controller.go

```diff
@@ -762,3 +794,27 @@ func backupStorageName(repoName string, cluster *everestv1alpha1.DatabaseCluster
 	}
 	return cluster.Spec.Backup.Schedules[scheduleInd-2].BackupStorageName, nil
 }
+
+// handleCleanup handles the cleanup of the DatabaseClusterBackup.
+func (r *DatabaseClusterBackupReconciler) handleStorageCleanup(
+	ctx context.Context,
+	dbcBackup *everestv1alpha1.DatabaseClusterBackup,
+	childBackup client.Object,
```

Comment:
The name `childBackup` seems to be a bit confusing. For on-demand backups it's indeed like a child - first we create `dbb` CR then the upstream backup CR after it. But in case of scheduled backups the first one appeared is the upstream backup and then the `dbb` CR is created after it. 
So should it be something without the hierarchy in the name, e.g. `upstreamBackup`?

Reply:
Sure, `upstreamBackup` sounds better. Updated!